testing/tests/ui/caller_arguments.stderr

```diff
@@ -95,3 +57,27 @@ error: failed to parse template source
 100 | |     {%- endcall -%}
 101 | |     "#,
     | |______^
+
+error[E0599]: no method named `caller` found for reference `&CallerInCaller` in the current scope
```

Comment:
That doesn't seem like the expected output. ^^'

Reply:
This is the error message by the official `jinja2` engine:
```
No caller defined
```

What would you suggest to do?